addons/product/models/product_attribute.py

```diff
@@ -18,12 +18,29 @@ class ProductAttribute(models.Model):
     sequence = fields.Integer('Sequence', help="Determine the display order", index=True)
     attribute_line_ids = fields.One2many('product.template.attribute.line', 'attribute_id', 'Lines')
     create_variant = fields.Selection([
-        ('no_variant', 'Never'),
-        ('always', 'Always'),
-        ('dynamic', 'Only when the product is added to a sales order')],
+        ('always', 'Instantly'),
+        ('dynamic', 'Dynamically'),
+        ('no_variant', 'Never')],
         default='always',
-        string="Create Variants",
-        help="Check this if you want to create multiple variants for this attribute.", required=True)
+        string="Variants Creation Mode",
+        help="""Select the variants creation mode:
```

Comment:
I think we can remove this line in fact.

Reply:
Done